python/ray/_raylet.pyx

```diff
@@ -4051,6 +4058,12 @@ cdef class CoreWorker:
             for fd in function_descriptors:
                 self.fd_to_cgname_dict[fd] = cg_name
 
+    def get_thread_pool_for_async_event_loop(self):
+        if self.thread_pool_for_async_event_loop is None:
+            self.thread_pool_for_async_event_loop = ThreadPoolExecutor(
+                max_workers=int(os.getenv("RAY_ASYNC_THREAD_POOL_SIZE", 1)))
```

Comment:
Q: maybe I should always use 1 thread? I used multiple for some exploration. I think when there are lots of async tasks, it may have some perf benefits

Reply:
So it's thread safe to use multiple threads? Before it's done  in main thread so it's 1.

Reply:
Hmm actually a good point. I think it should be thread-safe, but not 100% sure. Maybe I will just use 1 thread for now